packages/ifpack2/src/Ifpack2_Details_FastILU_Base_def.hpp

```diff
@@ -448,6 +451,7 @@ Params::getDefaults()
   p.blockSize = 1;      // # of rows / thread, for SpTRSV
   p.blockCrs = false;   // whether to use block CRS for fastILU
   p.blockCrsSize = 1;   // block size for block CRS
+  p.fillBlocks = false; // whether input matrix needs to be filled
```

Comment:
I believe the input matrix will be filled by default. Adding zeroes is just a convenience.

Reply:
@srajama1 , yeah, filling with zeroes is off by default. It was useful for testing against unblocked CRSs.